.changelog/2394.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/mongodbatlas_search_index: Returns error if the `analyzers` attribute contains unknown fields.
```

Comment:
```suggestion
resource/mongodbatlas_search_index: Returns error if the `analyzers` attribute contains unknown fields
```

no dot at the end, see: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md